src/queries/fei-goals.sql

```diff
@@ -0,0 +1,107 @@
+* SELECT SET_CONFIG('jdi.createdbetween','["2023-10-01","2023-10-15"]',TRUE);
+*/
+with bad as (
+	SELECT *
+	FROM public."Goals"
+	where name != '(FEI) The recipient will eliminate and/or reduce underenrollment as part of the Full Enrollment Initiative (as measured by monthly reported enrollment)'
+	AND name like '%monthly reported enrollment%'
+	and COALESCE("goalTemplateId",0) != 19017
+	AND id not in (52460, 52461, 52462, 52463, 52248, 52251, 52249, 52250, 55244, 55172, 55908, 55420, 55421, 56033, 50565, 50613, 50612, 50343, 50614)
+	UNION
+	SELECT *
+	FROM public."Goals"
+	where name = '(FEI) The recipient will eliminate and/or reduce underenrollment as part of the Full Enrollment Initiative (as measured by monthly reported enrollment)'
+	and COALESCE("goalTemplateId",0) != 19017
+	UNION
+	SELECT *
+	FROM public."Goals"
+	where name != '(FEI) The recipient will eliminate and/or reduce underenrollment as part of the Full Enrollment Initiative (as measured by monthly reported enrollment)'
+	AND name like '(FEI)%'
+	and COALESCE("goalTemplateId",0) != 19017
+	UNION
+	SELECT *
+	FROM  public."Goals"
+	WHERE "goalTemplateId" = 19017
+		)
+		select
+			r.name "recipient",
+			gr.number "grant number",
+			gr."regionId",
+			b.id "goalId",
+			b."goalTemplateId",
+			b.name "goal text",
+			b."createdAt",
+			b.status,
+			b."createdVia",
+			b."onApprovedAR",
+			gfr.response
+		FROM bad b
+		LEFT JOIN "GoalFieldResponses" gfr
+		ON b."id" = gfr."goalId"
+		JOIN "Grants" gr
+		ON b."grantId" = gr.id
+		JOIN "Recipients" r
+		ON gr."recipientId" = r.id
+		WHERE (NULLIF(current_setting('jdi.regionIds', true), '') IS NULL
+			   OR gr."regionId" in (
+				   SELECT value::integer AS my_array
+			       FROM json_array_elements_text(COALESCE(NULLIF(current_setting('jdi.regionIds', true), ''),'[]')::json) AS value
+			   ))
+		AND (NULLIF(current_setting('jdi.recipients', true), '') IS NULL
+			   OR r.name in (
+				   SELECT value::text AS my_array
+			       FROM json_array_elements_text(COALESCE(NULLIF(current_setting('jdi.recipients', true), ''),'[]')::json) AS value
+			   ))
+		AND (NULLIF(current_setting('jdi.grantNumber', true), '') IS NULL
+			   OR gr.number in (
+				   SELECT value::text AS my_array
+			       FROM json_array_elements_text(COALESCE(NULLIF(current_setting('jdi.grantNumber', true), ''),'[]')::json) AS value
+			   ))
+		AND (NULLIF(current_setting('jdi.goal', true), '') IS NULL
+			   OR b.name in (
+				   SELECT value::text AS my_array
+			       FROM json_array_elements_text(COALESCE(NULLIF(current_setting('jdi.goal', true), ''),'[]')::json) AS value
+			   ))
+		AND (NULLIF(current_setting('jdi.status', true), '') IS NULL
+			   OR b.status in (
+				   SELECT value::text AS my_array
+			       FROM json_array_elements_text(COALESCE(NULLIF(current_setting('jdi.status', true), ''),'[]')::json) AS value
+			   ))
+		AND (NULLIF(current_setting('jdi.createdVia', true), '') IS NULL
```

Comment:
Each one of the blocks seems to follow the same pattern for different fields can you put a comment at the top just explaining what this is doing?